### PR TITLE
[zh-cn]: update the translation of Navigator: clipboard property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/navigator/clipboard/index.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
 slug: Web/API/Navigator/clipboard
+l10n:
+  sourceCommit: 7087ffd50a4d81d1b91fe603c26456e9ce398574
 ---
 
-剪贴板 [Clipboard API](/zh-CN/docs/Web/API/Clipboard_API) 为 **{{domxref("Navigator")}}** 接口添加了只读属性 **`clipboard`**，该属性返回一个可以读写剪切板内容的 {{domxref("Clipboard")}} 对象。在 Web 应用中，剪切板 API 可用于实现剪切、复制、粘贴的功能。
+{{APIRef("Clipboard API")}} {{securecontext_header}}
 
-只有在用户事先授予网站或应用对剪切板的访问许可之后，才能使用异步剪切板读写方法。许可操作必须通过取得权限 [Permissions API](/zh-CN/docs/Web/API/Permissions_API) 的 `"clipboard-read"` 和/或 `"clipboard-write"` 项获得。
+{{domxref("Navigator")}} 接口的只读属性 **`clipboard`** 返回一个 {{domxref("Clipboard")}} 对象，用于读取和写入剪贴板的内容。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/clipboard/index.md:10]: could be more brief :)

```suggestion
{{domxref("Navigator")}} 接口的只读属性 **`clipboard`** 返回一个用于读写剪贴板内容的 {{domxref("Clipboard")}} 对象。
```

File: files/zh-cn/web/api/navigator/clipboard/index.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
 
-```js
-theClipboard = navigator.clipboard;
-```
+## 值
 
-### 值
-
-用于访问系统剪切板的 {{domxref("Clipboard")}} 对象。
+{{domxref("Clipboard")}} 对象用于访问系统剪贴板。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/clipboard/index.md:16]: This describe the return value, so it's better to use:

```suggestion
用于访问系统剪贴板的 {{domxref("Clipboard")}} 对象。
```

File: files/zh-cn/web/api/navigator/clipboard/index.md [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
 
 ## 示例
 
-以下代码使用 `navigator.clipboard` 来访问系统剪切板，以读取系统剪切板的内容。
+以下代码使用了 `navigator.clipboard` 访问系统剪贴板，以便读取剪贴板中的文本内容。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/clipboard/index.md:20]:
```suggestion
以下代码使用 `navigator.clipboard` 来访问系统剪贴板，以便读取剪贴板中的文本内容。
```

File: files/zh-cn/web/api/navigator/clipboard/index.md [hunk 4]
```diff
@@ -29,9 +27,9 @@ navigator.clipboard
   );
 ```
 
-这个代码片段将 HTML 中拥有类名 `"cliptext"` 的第一个元素的内容替换为剪切板中的内容。这段代码可用于在浏览器拓展中定时自动更新或者由事件触发，实时显示当前剪切板上的内容。
+该代码片段将层叠样式表为 `“cliptext”` 的元素的内容替换为剪贴板中的文本内容。这段代码可能用于浏览器扩展程序，该扩展程序显示当前剪贴板的内容，并会定期自动更新或在特定事件触发时更新。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/clipboard/index.md:30]:
```suggestion
该代码片段将设置了 `"cliptext"` 类的元素的内容替换为剪贴板中的文本内容。这段代码可能用于浏览器扩展程序，该扩展程序显示当前剪贴板的内容，并会定期或在特定事件触发时自动更新。
```

File: files/zh-cn/web/api/navigator/clipboard/index.md [hunk 5]
```diff
@@ -29,9 +27,9 @@ navigator.clipboard
...
 
-如果剪切板为空，或者不包含文本，则 `"cliptext"` 元素的内容将被清空。这是因为在剪切板为空或者不包含文本时，{{domxref("Clipboard.readText", "readText()")}} 会返回一个空字符串。
+如果剪贴板为空或不包含文本，则清除 `“cliptext”` 元素的内容。这是因为如果剪贴板为空或不包含文本，否则 {{domxref("Clipboard.readText", "readText()")}} 会返回空字符串。
```
Comment (yin1999) [inline files/zh-cn/web/api/navigator/clipboard/index.md:32]: Please do not replace quotes in inline code with full-width quotes as it represents a JavaScript/CSS/HTML string.

```suggestion
如果剪贴板为空或不包含文本，则清除 `"cliptext"` 元素的内容。这是因为如果剪贴板为空或不包含文本，{{domxref("Clipboard.readText", "readText()")}} 会返回空字符串。
```

Comment (T34-active) [reply files/zh-cn/web/api/navigator/clipboard/index.md:32]: OK

**Review by yin1999 — APPROVED** [review]
LGTM :+1:

**Pull request merged**